patches/ui-webui-resources-tools-optimize_webui.py.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/ui/webui/resources/tools/optimize_webui.py b/ui/webui/resources/tools/optimize_webui.py
+index 6b5978eeb4145..1242551f785e7 100755
+--- a/ui/webui/resources/tools/optimize_webui.py
++++ b/ui/webui/resources/tools/optimize_webui.py
+@@ -32,6 +32,7 @@ for excluded_file in [
+     'resources/js/cr.js',  # This file relies on globals.
+     'resources/js/load_time_data.js',
+     'resources/ash/common/load_time_data.m.js',
++    'resources/brave/leo/bundle.js',
```

Comment:
pls implement this as a chromium_src inline python file. See an example here: https://github.com/brave/brave-core/pull/17882/files#diff-f01344c9cc552faeb1b0a06e28821496758e566b40b29aedd83829cb683da1d0R17

basically you need to:
1. add a python file at the same path in `chromium_src` where an upstream file is located with a content like this:
```
_BASE_EXCLUDES.extend([
  'chrome://resources/brave/leo/bundle.js',
  '//resources/brave/leo/bundle.js',
])
```
2. inline it with a single-line patch right before the `__main__` check:
```
+ from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/ui/webui/resources/tools/optimize_webui.py", globals(), locals())
ui/webui/resources/tools/optimize_webui.py
if __name__ == '__main__':
  main(sys.argv[1:])
```

Reply:
Cool, thanks @goodov!